README.md

```diff
@@ -42,9 +42,10 @@ git clone git@github.com:bonitoo-io/telegraf-input-db2.git
 __Modify telegraf configuration__
 
 ```toml
-[[inputs.execd]]
-  command = ["/path/to/telegraf-input-db2/db2_metrics.sh"]
-  signal = "none"
+[[inputs.exec]]
+  commands = ["/path/to/telegraf-input-db2/db2_metrics.sh"]
+  timeout = "60s"
+  data_format = "influx"
```

Comment:
Shouldn't we add a commented `interval` property to make clear how frequently it is executed?

```suggestion
  data_format = "influx"
#   ## Execution interval, can override default interval settings in [agent] section
#   interval = "60s"  
```
It is already described this way in [inputs/ceph plugin](https://github.com/influxdata/telegraf/blob/master/plugins/inputs/ceph/ceph.go#L52)

Reply:
Nice!